pyproject.toml

```diff
@@ -5,6 +5,7 @@ build-backend = "hatchling.build"
 [project]
 name = "schwifty"
 dynamic = ["version"]
+description = "IBAN parsing and validation"
```

Comment:
```suggestion
description = "SWIFT network IBAN parsing and validation"
```

Reply:
Refs https://en.wikipedia.org/wiki/SWIFT

Reply:
I copied the description from GitHub. I‘d rather not deviate from that.

If you want that change, better apply it everywhere?

Also, SWIFT is only mentioned in an aside about the name, can‘t find it elsewhere with a quick glace. Doesn‘t seem to be that important then?

Reply:
I'm also okay with the description without specifically mentioning SWIFT